Sources/AppcuesKit/Presentation/Traits/TraitMetadataDelegate.swift

```diff
@@ -43,8 +43,10 @@ public class TraitMetadataDelegate: NSObject {
 
     public func registerHandler(for key: String, animating: Bool, observer: @escaping (TraitMetadata) -> Void) {
         if animating {
+            nonAnimatingSubscribers.removeValue(forKey: key)
```

Comment:
seems like this would always be the desired automatic behavior when re-registering the same key. Only thought is maybe the ever so slightly more DRY version of calling `removeHandler(for: key)` once at the top of the function? not important though.

Reply:
Updated to use `removeHandler` while rebasing 👍